### PR TITLE
add one more jack

### PR DESCRIPTION
**Pull request by escapiststupor**
單純是練習一下往upstream發pr的功能怎麼用

### CODE REVIEW

File: packages/dailyjack-core/jacks.json [hunk 1]
```diff
@@ -225,5 +225,14 @@
...
+    "id": "28",
+    "title": "鐵達尼號之所以會沉船",
+    "contents": [
+      "是因為 Jack 把螺絲弄鬆了"
+    ],
+    "author": "不是我",
+    "isLimited": true
   }
-]
+]
```
Comment (kevin940726) [inline packages/dailyjack-core/jacks.json]: Please preserve the end-of-file line.

File: packages/dailyjack-core/jacks.json [hunk 2]
```diff
@@ -225,5 +225,14 @@
...
     ],
     "author": "scars"
+  },
+  {
+    "id": "28",
+    "title": "鐵達尼號之所以會沉船",
+    "contents": [
+      "是因為 Jack 把螺絲弄鬆了"
+    ],
+    "author": "不是我",
```
Comment (kevin940726) [inline packages/dailyjack-core/jacks.json]: Please acknowledge yourself.

File: packages/dailyjack-core/jacks.json [hunk 3]
```diff
@@ -225,5 +225,14 @@
...
+    "isLimited": true
```
Comment (kevin940726) [inline packages/dailyjack-core/jacks.json]: This is not a limited jack.

**Review by kevin940726 — CHANGES_REQUESTED** [review]

File: packages/dailyjack-core/jacks.json (hunk 2 above)
Comment (escapiststupor) [reply packages/dailyjack-core/jacks.json]: 會不會被吉QQ

File: packages/dailyjack-core/jacks.json (hunk 3 above)
Comment (escapiststupor) [reply packages/dailyjack-core/jacks.json]: limited Jack會發生什麼事

Comment (escapiststupor) [reply packages/dailyjack-core/jacks.json]: 我懂了....

File: packages/dailyjack-core/jacks.json (hunk 2 above)
Comment (kevin940726) [reply packages/dailyjack-core/jacks.json]: Don't worry 😂

**Review by kevin940726 — APPROVED** [review]

**Pull request merged**